scripts/perfmon

```diff
@@ -1103,7 +1103,7 @@ def main():
             print_debug("sr_uuid_list = %s" % sr_uuid_list)
 
             # Remove monitors for SRs no longer listed in the rrd_updates page
-            for uuid in sr_mon_lookup:
+            for uuid in list(sr_mon_lookup.keys()):
```

Comment:
Please add a comment for readers to make it clear why list(dict_var.keys()) is needed here:
```suggestion
            # we use .pop() inside the loop, use list(dict_var.keys()):
            for uuid in list(sr_mon_lookup.keys()):
```